### PR TITLE
Fixes 3D models being omitted from GLTF export since switch to GLTFast

### PR DESCRIPTION
**Pull request by andybak**

### CODE REVIEW

File: Assets/Scripts/Export/ImportMaterialCollector.cs [hunk 1]
```diff
@@ -92,6 +92,25 @@ public void AddSvgIem(Material unityMaterial)
   }
 #endif
 
+        // Used for GLTFast
+        public void Add(Material unityMaterial, bool transparent)
```
Comment (mikeage) [inline Assets/Scripts/Export/ImportMaterialCollector.cs:96]: Did you mean to do something with `transparent`?

Comment (andybak) [reply Assets/Scripts/Export/ImportMaterialCollector.cs]: Nope. That was a leftover from a last minute change. It's now inferred directly from the material's renderQueue value

**Review by mikeage — APPROVED** [review]
LGTM

**Comment by andybak** [comment]
n.b. I suspect there are other issues with exports that have 3d models but this is a step forward.

**Comment by mikeage** [comment]
> n.b. I suspect there are other issues with exports that have 3d models but this is a step forward.

Should we keep this open for more fixes, or merge it to get it out to whoever's testing this (is someone using it?)

**Comment by andybak** [comment]
@mikeage :
> Should we keep this open for more fixes, or merge it to get it out to whoever's testing this (is someone using it?)

Oh - definitely merge. This fixes a bug that masks other bugs so will increase the chance of other issues being reported.

**Pull request merged**